dataHandler/dataProviders.json

```diff
@@ -28,6 +28,6 @@
 
   "call_symbol_abbreviation": "C",
   "put_symbol_abbreviation": "P",
-  "date_time_format": "%m/%d/%Y",
+  "date_time_format": "%m/%d/%y",
```

Comment:
Please note that this won't work with the purchased data from iVolatility, but it will work with the test data.